doku/tests/printers/enum/of_tag/internal/of_fields/named/output.without-user-comments-wrapped.separated.toml

```diff
@@ -0,0 +1,8 @@
+[value]
```

Comment:
I'm not sure i like the pseudo-xml syntax but i agree that the current approach is confusing. I guess that it's a good solution.

In the example from `TestSpecV2` shouldn't the `[[steps]]` label be repeated for each variant since it's an array?

Reply:
Yes, for clarity the `[[steps]]` label should be repeated, I think;

So currently `doku::to_toml::<TestSpecV2>()` returns:
```toml
[[steps]]
type = "AssertContract"
cid = "string"
# Path to the expected ion file
ion = "string"
# Path to the expected esl file; optional
esl = "string"
# or
cid = "string"
# Assert whether the contract was deleted
deleted = true

# or
type = "AssertRecs"
# URL to the maintenance api of rabbitmq. Default: "localhost:15672"
url = "string"
# Queue for the test. Usually it is a random value, e:g '{{ unique }}'
queue = "string"
# Path to the expected file
file = "string"

...
```
... would it be possible to slightly adjust it, so that what we have is this?
```toml
[[steps]]
type = "AssertContract"
cid = "string"
# Path to the expected ion file
ion = "string"
# Path to the expected esl file; optional
esl = "string"

# or

[[steps]]
type = "AssertContract"
cid = "string"
# Assert whether the contract was deleted
deleted = true

# or

[[steps]]
type = "AssertRecs"
# URL to the maintenance api of rabbitmq. Default: "localhost:15672"
url = "string"
# Queue for the test. Usually it is a random value, e:g '{{ unique }}'
queue = "string"
# Path to the expected file
file = "string"
```
(that being the default formatting, also)